pipelines/mnist-pipelines/mnist_pipeline.py

```diff
@@ -65,15 +65,15 @@ def mnist_pipeline(model_export_dir='gs://your-bucket/export',
   serve = dsl.ContainerOp(
       name='serve',
       image='gcr.io/ml-pipeline/ml-pipeline-kubeflow-deployer:'
-            '7775692adf28d6f79098e76e839986c9ee55dd61',
+            '8dfa2d9f832e5291649e3bc9d58468da7c6ed92a',
       arguments=serve_args
   )
   serve.after(train)
 
 
   webui_args = [
           '--image', 'gcr.io/kubeflow-examples/mnist/web-ui:'
-                     'v20190304-v0.2-176-g15d997b-pipelines',
+                     'latest',
```

Comment:
Can we pin to a specific commit rather than using latest? 
The semantics of latest are to always point to the newest image which is brittle